chain/actors/policy/policy.go.template

```diff
@@ -63,14 +63,23 @@ func AddSupportedProofTypes(types ...abi.RegisteredSealProof) {
                 miner{{.}}.PreCommitSealProofTypesV7[t] = struct{}{}
                 miner{{.}}.PreCommitSealProofTypesV7[t+abi.RegisteredSealProof_StackedDrg2KiBV1_1] = struct{}{}
                 miner{{.}}.PreCommitSealProofTypesV8[t+abi.RegisteredSealProof_StackedDrg2KiBV1_1] = struct{}{}
-            {{else}}
+            {{else if (eq . 5)}}
                 miner{{.}}.PreCommitSealProofTypesV8[t+abi.RegisteredSealProof_StackedDrg2KiBV1_1] = struct{}{}
                 wpp, err := t.RegisteredWindowPoStProof()
                 if err != nil {
                     // Fine to panic, this is a test-only method
                     panic(err)
                 }
 
+                miner{{.}}.WindowPoStProofTypes[wpp] = struct{}{}
+            {{else}}
+                miner{{.}}.PreCommitSealProofTypesV8[t+abi.RegisteredSealProof_StackedDrg2KiBV1_1] = struct{}{}
+                wpp, err = t.RegisteredWindowPoStProof()
+                if err != nil {
+                    // Fine to panic, this is a test-only method
+                    panic(err)
+                }
+
```

Comment:
How is this different from the v5 thing?

Reply:
for v5 we := define `wpp` and `err` for the first time, v6 onwards uses the same variables